src/main/java/Products.java

```diff
@@ -0,0 +1,59 @@
+public class Products {
+
+    private final String productName;
+    private final double productPrice;
+    private final boolean exempt;
+    private final boolean imported;
+    private final int productID;
+
+    private Products(int productID, String productName, double productPrice, boolean exempt, boolean imported) {
+        this.productID = productID;
+        this.productName = productName;
+        this.productPrice = productPrice;
+        this.exempt = exempt;
+        this.imported = imported;
+    }
+
+    public static final Products book = new Products(1,"Book",12.49, true, false);
```

Comment:
static final could be all UPPERCASE. (I know there are multiple schools of thought on this but I must be old fashioned... what can I do???

Reply:
This is how intellij autofills.

Reply:
Ahhh never on an IDE lol!